feat(ipam): add info for reserving IPs (#3678)
scaleway/docs-content: network/ipam/how-to/manage-ips.mdx

- Viewing the details of a private IP address, including its ID, tags, creation date and to which resource (if any) it is attached
- Editing the tags of a private IP address

> To reserve a private IP address, see the [dedicated document](/network/ipam/how-to/reserve-ip/).

Reviewer: ```suggestion
Learn [how to reserve a private IP address](/network/ipam/how-to/reserve-ip/).
```
